src/documentation/pages/components/forms/range-slider/examples/_radio_slider.vue

```diff
@@ -3,9 +3,9 @@
     .p-text(slot="example-description" class="-mb--3 -text")
       | Enable range sliders to function like radio buttons.
     .chi-form__item(slot="example")
-      label(class="chi-label", for="range07") Range label
+      label(class="chi-label", id="range-label", for="range07") Range label
```

Comment:
no, what i meant is that the `input` is what is missing its id, so that label can reference it with `for`. Label itself doesn't need id